docs/integrations/delta-lake-dagster.md

```diff
@@ -0,0 +1,278 @@
+
+## Working with Partitioned Assets
+Partitioning is an important feature of Delta Lake that can make your computations more efficient. The Delta Lake I/O manager helps you read and write partitioned data easily. You can work with static partitions, time-based partitions, multi-partitions, and dynamic partitions.
+
+For example, you can partition the Iris dataset on the `species` column as follows:
+
+```
+import pyarrow as pa
+
+from dagster import StaticPartitionsDefinition, asset
+
+@asset(
+  partitions_def=StaticPartitionsDefinition(
+      ["Human", "Horse",]
+  ),
+  metadata={"partition_expr": "n_legs"},
+)
+def dataset_partitioned(
+   context,
+   clean_dataset: pa.Table,
+   ) -> pa.Table:
+   animals = context.asset_partition_key_for_output()
+   table = clean_dataset
+
+   return table.filter(pc.field("animals") == animals)
+```
+
+To partition your data, make sure to include the relevant `partitions_def` and `metadata` arguments to the `@asset` decorator. Refer to the Dagster documentation on [partitioning assets](https://docs.dagster.io/concepts/partitions-schedules-sensors/partitioning-assets) for more information.
+
+## Using Delta Lake and Dagster with Pandas
+To read and write data to Delta Lake using pandas, use the `DeltaLakePandasIOManager()`. 
+
+You will need to install it using:
+
+```
+pip install dagster-deltalake-pandas
+```
+
+In your `Definitions` object, change the `io_manager` to `DeltaLakePandasIOManager()`:
+
+```
+from dagster_deltalake_pandas import DeltaLakePandasIOManager
+
+
+defs = Definitions(
+   assets=all_assets,
+   resources={
+        "io_manager": DeltaLakePandasIOManager(
+            root_uri="path/to/deltalake",
+            storage_options=LocalConfig(),
+            schema="dagster_deltalake",
+        ),
+   },
+)
+```
+
+Now you can read and write Dagster Assets defined as pandas DataFrames in Delta Lake format. For example:
+
+```
+import pandas as pd
+from dagster import asset
+
+@asset
+def iris_dataset() -> pd.DataFrame:
+   return pd.read_csv(
+       "https://docs.dagster.io/assets/iris.csv",
+       names=[
+           "sepal_length_cm",
+           "sepal_width_cm",
+           "petal_length_cm",
+           "petal_width_cm",
+           "species",
+       ],
+   )
+```
+
+## Using Delta Lake and Dagster with Polars
+To read and write data to Delta Lake using pandas, use the `DeltaLakePolarsIOManager()`. 
+
+You will need to install it using:
+
+```
+pip install dagster-polars
+```
+
+In your `Definitions` object, change the `io_manager` to `DeltaLakePolarsIOManager()`:
+
+```
+from dagster_polars import PolarsDeltaIOManager
+
+defs = Definitions(
+   assets=all_assets,
+   resources={
+        "io_manager": PolarsDeltaIOManager(
```

Comment:
@avriiil this is the io manager from dagter-polars, this implementation is relying on UPathIOmanager instead of DBIOManager, I would say its better to change this to DeltaLakePolarsIOManager

Reply:
Thanks @ion-elgreco!